features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml

```diff
@@ -16,22 +16,22 @@
 ~ under the License.
 -->
 <Scopes>
-    <Scope id="openid">
+    <Scope id="openid" displayName="Open ID" description="Indicate the use of OpenID Connect protocol.">
         <Claim>
             sub,email,email_verified,name,family_name,given_name,middle_name,nickname,preferred_username,upn,groups,profile,picture,website,gender,birthdate,zoneinfo,locale,updated_at,phone_number,phone_number_verified,address,street_address,country,formatted,postal_code,locality,region
         </Claim>
     </Scope>
-    <Scope id="email">
+    <Scope id="email" displayName="Email" description="View the e-mail address of the user.">
         <Claim>email,email_verified</Claim>
     </Scope>
-    <Scope id="profile">
+    <Scope id="profile" displayName="Profile" description="Retrieve profile information of the user.">
         <Claim>name,family_name,given_name,middle_name,nickname,preferred_username,profile,picture,website,gender,birthdate,zoneinfo,locale,updated_at
         </Claim>
     </Scope>
-    <Scope id="phone">
+    <Scope id="phone" displayName="Phone" description="View the phone number of the user.">
```

Comment:
This returns the `phone_number_verified` as well. 
   So "View information related to phone number of the user." will be more appropriate?

Reply:
+1